conf/etc/nginx/sites-available/default

```diff
@@ -17,16 +17,22 @@
 # Please see /usr/share/doc/nginx-doc/examples/ for more detailed examples.
 ##
 
+# Redirect to HTTPS
 server {
 	listen 80 default_server;
-	listen [::]:80 default_server ipv6only=on;
+	listen [::]:80 default_server;
+	server_name localhost;
+	return 301 https://$server_name$request_uri;
+}
+# HTTPS server
+#
+server {
+	listen 443;
+	server_name localhost ipv6only=on;
 
 	root /opt/piecewise_web;
 	index index.html index.htm;
 
-	# Make site accessible from http://localhost/
-	server_name localhost;
-
     location = /stats { rewrite ^ /stats/; }
```

Comment:
nit: it looks like there are some indention inconsistencies here (and throughout the file). It would be good if this PR could clean up the indention (that may have already existed before).